javagen/src/main/java/com/azure/autorest/model/javamodel/JavaPackage.java

```diff
@@ -165,19 +171,47 @@ public final void addModuleInfo(ModuleInfo moduleInfo) {
         addJavaFile(javaFile);
     }
 
+    public final void addChangelog() {
+        PlainFile plainFile = new PlainFile("CHANGELOG.md");
+        plainFile.loadFromResource("changelog.md", settings.getLlcProperties());
+        addPlainFile(plainFile);
+    }
+
+    public final void addServicePom() {
+        PlainFile plainFile = new PlainFile("pom.xml");
```

Comment:
Should this be an xml file?

Reply:
It can be. But I load this pom from resources folder. I add necessary methods to `PlainFile`. I don't care filename suffix.